modules/lighthouse_service.js

```diff
@@ -31,6 +31,20 @@ const createPlatesFromBarcodes = async (moduleOptions) => {
   return responses
 }
 
+// Accepts a list of plateBarcodes in the moduleOptions
+// Send a GET request to the Lighthouse service API
```

Comment:
Sharing my recently improved javascript knowledge here but if you do 

`const findPlatesFromBarcodes = async ({ barcodes }) => {
`
then you can do:

`params: { barcodes }`

Just saves you having to explicity name the variable as JavaScript infers it.

Reply:
That would mean I needed to do the inverse with the response from getPlatesFromBoxBarcodes though, at the moment I can just pass the response straight through.